content/docs/2.8/scalers/cpu.md

```diff
@@ -30,11 +31,34 @@ triggers:
 - `value` - Value to trigger scaling actions for:
 	- When using `Utilization`, the target value is the average of the resource metric across all relevant pods, represented as a percentage of the requested value of the resource for the pods.
 	- When using `AverageValue`, the target value is the target value of the average of the metric across all relevant pods (quantity).
+- `containerName` - Set this to target the specific container rather than the entire pod. Defaults to empty if not specified.
```

Comment:
```suggestion
- `containerName` - Name of the specific container to scaled based on, rather than the entire pod. Defaults to empty if not specified.
```

Reply:
`to scaled based on` ?

Reply:
`to scale on` was more what I meant or ´to scale based on its CPU` maybe?

Reply:
`to scale based on its CPU` is good.

Reply:
and the same for memory docs

Reply:
suggestion updated